dash-renderer/src/actions/index.js

```diff
@@ -176,7 +183,7 @@ function reduceInputIds(nodeIds, InputGraph) {
     /*
      * Create input-output(s) pairs,
      * sort by number of outputs,
-     * and remove redudant inputs (inputs that update the same output)
```

Comment:
re-dud-ant
*adjective*
- Equally bodacious and/or gnarly as the previous dude.